src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
-You can obtain this information from the "Settings" tab of the server page in the Outline Manager.
-Alternatively, you can check the 'access.txt' file under the '/opt/outline' directory of an Outline server. An example apiUrl is: https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z.
-
-See [Full API Documentation](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml).
-The OpenAPI specification can be found at [api.yml](./server/api.yml).
+1. **Find your `apiUrl`:** (e.g. `https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z`)
 
-### Examples
+   - **Deployed with the Installation Script:** Run `grep "apiUrl" /opt/outline/access.txt | cut -d: -f 2-`
 
-Start by storing the apiURL you see see in that file, as a variable. For example:
+   - **Deployed with the Outline Manager:** Check the "Settings" tab.
 
-```
-API_URL=https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z
-```
+   - **Local Deployments from Source:** `https://[::]:8081/TestApiPrefix`
 
-You can then perform the following operations on the server, remotely.
+1. **API Examples:**
 
-List access keys
+   Replace `$API_URL` with your actual `apiUrl`.
 
-```
-curl --insecure $API_URL/access-keys/
-```
+   - **List access keys:** `curl --insecure $API_URL/access-keys/`
 
-Create an access key
+   - **Create an access key:** `curl --insecure -X POST $API_URL/access-keys`
 
-```
-curl --insecure -X POST $API_URL/access-keys
-```
+   - **Get an access key (e.g. ID 1):** `curl --insecure $API_URL/access-keys/1`
 
-Get an access key (e.g. get access key 1)
+   - **Rename an access key:** `curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name`
 
-```
-curl --insecure $API_URL/access-keys/1
-```
+   - **And more...**
 
-Rename an access key
-(e.g. rename access key 2 to 'albion')
+1. **Further Options:**
 
-```
-curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name
-```
+   Consult the [OpenAPI spec](./server/api.yml) and [documentation](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml) for more options.
 
-Remove an access key
-(e.g. remove access key 2)
-
-```
-curl --insecure -X DELETE $API_URL/access-keys/2
-```
+## Testing
 
-Set a data limit for all access keys
-(e.g. limit outbound data transfer access keys to 1MB over 30 days)
+### Manual
 
-```
-curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"limit": {"bytes": 1000}}' $API_URL/experimental/access-key-data-limit
-```
+1. Prerequisites
 
-Remove the access key data limit
+   - A locally built Docker image containing your modifications.
+   - Your Docker image uploaded to a registry (e.g., Docker Hub, Quay.io).
 
-```
-curl -v --insecure -X DELETE $API_URL/experimental/access-key-data-limit
-```
+1. Setup
 
-## Testing
+   - **Environment Variable:** Set `SB_IMAGE` to the uploaded image location. Example:
```

Comment:
Move this to after start service. This is optional, only if you need to customize the image name (uncommon)

Reply:
Don't you need this set for [`start.action.sh`](https://github.com/Jigsaw-Code/outline-server/blob/master/src/shadowbox/docker/start.action.sh#L50) to pick up the image name customization? I thought that was the whole point of this manual testing section of a custom image.